### DISCUSSION TITLE
Over 1MB limit?

### DISCUSSION THREAD

**Discussion by Kayqubad** [0x7c13/Notepads#1028] [Ideas]
Do you have any plans of allowing to open files above 1mb?
It's been since we've been stuck with the limitation.

Other than that, thanks for the amazing app!

**Comment by 0x7c13** [comment]
Not possible for now by the current design but I totally understand your pain point here. Mayber in a future version of WinUI, MSFT will offer a better rich edit control for us to consume and that could potentially solve the problem.
